src/Console/SshCommand.php

```diff
@@ -49,7 +49,10 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $host = $this->deployer->hosts->get($input->getArgument('hostname'));
+        $firstHostname = $this->deployer->hosts->getIterator()->current()->getHostname();
```

Comment:
I don't quite understand the `dep ssh - interactive`path.
User is typing `dep ssh - interactive`and get a list of hosts?

That would be an enhancement for another pull request. Would it fit with an additional "check if there is only one host registered"?